slackbot.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"log"
```

Comment:
use `gravitational/logrus` to handle logging here and below

Reply:
Changed.